extensions/websockets-next/deployment/src/main/java/io/quarkus/websockets/next/deployment/WebSocketProcessor.java

```diff
@@ -405,7 +410,8 @@ public String apply(String name) {
     @BuildStep
     public void registerRoutes(WebSocketServerRecorder recorder, HttpRootPathBuildItem httpRootPath,
             List<GeneratedEndpointBuildItem> generatedEndpoints, HttpBuildTimeConfig httpConfig, Capabilities capabilities,
-            BuildProducer<RouteBuildItem> routes) {
+            BuildProducer<RouteBuildItem> routes, List<EndpointHttpUpgradePolicyBuildItem> endpointHttpUpgradePolicies) {
```

Comment:
IIUIC the `EndpointHttpUpgradePolicyBuildItem` is not produced anywhere yet, or?

Reply:
It's produced in the `LocalHttpUpgradePolicyTest` test. It's kinda as with the `HttpUpgradePolicy` as we also do not produce it, just test it. I want to use it right after this PR.

If you mean that I shouldn't add it to this PR but later, sure I expected this opinion. I am ready to drop it if you require it, just confirm.

Reply:
> It's produced in the `LocalHttpUpgradePolicyTest` test. It's kinda as with the `HttpUpgradePolicy` as we also do not produce it, just test it. I want to use it right after this PR.

:+1: